### PR TITLE
Fixed minor misspelled words in the swift comments & Class name

### PR DESCRIPTION
**Pull request by byaruhaf**
Task/Issue URL: #1260, https://app.asana.com/0/414709148257752/1202716717386020/f
Tech Design URL:
CC:

**Description**:

This PR corrects a few minor misspelled words in the swift comments & Class name.

Example class:
```swift
class OnboardingDefaultBroswerViewController: OnboardingContentViewController {}
```
corrected to
```swift
class OnboardDefaultBrowserViewController: OnboardingContentViewController {}
```

Example comments:
```swift
        // Make sure muliline labels are correctly presented
```
corrected to 
```swift
        // Make sure multiline labels are correctly presented
```

In addition, this PR adjusts the expectation wait timeout, for the test  ```testWhenRequestIsMadeThenHasNavParameter``` by 1 second to correct the failing test. Is it acceptable to bump some of these timeouts within the scope of this PR?

### CODE REVIEW

File: DuckDuckGo/OnboardDefaultBrowserViewController.swift [hunk 1]
```diff
@@ -19,7 +19,7 @@
 import UIKit
 import Core
 
-class OnboardingDefaultBroswerViewController: OnboardingContentViewController {
+class OnboardDefaultBrowserViewController: OnboardingContentViewController {
```
Comment (samsymons) [inline DuckDuckGo/OnboardDefaultBrowserViewController.swift]: Good spotting on the `Browser` type – however we should keep the "Onboarding" part of the title. 🙂 

```suggestion
class OnboardingDefaultBrowserViewController: OnboardingContentViewController {
```

**Review by samsymons — CHANGES_REQUESTED** [review]
@byaruhaf Thanks so much for this! I spotted one thing to change, but otherwise this looks good.

By the way: we have a code freeze for the next couple weeks, so I've created a reminder task for myself internally to merge this once that is over.

**Comment by byaruhaf** [comment]
Thanks, @samsymons, 
I have accepted the correct for the one thing,😉
 I have a few other PR  I was working on for this repo so can be sending them through or wait for the freeze to end?

**Comment by samsymons** [comment]
@byaruhaf I would recommend sitting on those for 2 weeks or so. 🙂 But, if you want to discuss those sooner than that, please do open a GitHub Issue – we usually prefer to discuss larger changes on an issue first before anyone spends too long on a PR.

**Pull request closed without merging**